pkg/plugins/models.go

```diff
@@ -211,19 +211,21 @@ type PluginMetaDTO struct {
 }
 
 type DataSourceDTO struct {
-	ID              int64          `json:"id,omitempty"`
-	UID             string         `json:"uid,omitempty"`
-	Type            string         `json:"type"`
-	Name            string         `json:"name"`
-	PluginMeta      *PluginMetaDTO `json:"meta"`
-	URL             string         `json:"url,omitempty"`
-	IsDefault       bool           `json:"isDefault"`
-	Access          string         `json:"access,omitempty"`
-	Preload         bool           `json:"preload"`
-	Module          string         `json:"module,omitempty"`
-	JSONData        map[string]any `json:"jsonData"`
-	ReadOnly        bool           `json:"readOnly"`
-	AngularDetected bool           `json:"angularDetected"`
+	ID         int64          `json:"id,omitempty"`
+	UID        string         `json:"uid,omitempty"`
+	Type       string         `json:"type"`
+	Name       string         `json:"name"`
+	PluginMeta *PluginMetaDTO `json:"meta"`
+	URL        string         `json:"url,omitempty"`
+	IsDefault  bool           `json:"isDefault"`
+	Access     string         `json:"access,omitempty"`
+	Preload    bool           `json:"preload"`
+	Module     string         `json:"module,omitempty"`
+	JSONData   map[string]any `json:"jsonData"`
+	ReadOnly   bool           `json:"readOnly"`
+
+	AngularDetected        bool `json:"angularDetected"`
+	HideAngularDeprecation bool `json:"hideAngularDeprecation"`
```

Comment:
Yes, it makes sense. I tried avoiding it so we don't introduce a breaking change. However this piece of information is currently being used only by the sandbox and detect-angular-dashboards, as far as I know. Since we own both pieces, and it was suggested also in the original PR, I will do it the way you suggested :smile: